testdata/ingress_rg_notshared_alb/k8s/rg.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: my-route-group
+  annotations:
+    zalando.org/aws-load-balancer-shared: "false"
```

Comment:
This is still not showing up in the tests. Looks like the test is not capturing this detail. I'm not entirely sure if this is something that is not being tested or a configuration error.

Reply:
Seems to be a problem with the test.